core/schema/container.graphqls

```diff
@@ -557,6 +557,9 @@ type Container {
     """
     address: String!
   ): Container!
+
+  "The unique image reference"
```

Comment:
I think we should mention the caveats about this currently only working right after a `Container.From` in this doc string too.

Reply:
Thanks Eric @sipsma, fixed it.